core/src/main/scala/cats/syntax/parallel.scala

```diff
@@ -258,14 +269,45 @@ final class ParallelApOps[M[_], A](private val ma: M[A]) extends AnyVal {
     Parallel.parReplicateA(n, ma)
 }
 
-final class ParallelApplyOps[M[_], A, B](private val mab: M[A => B]) extends AnyVal {
+final class ParallelApOps1[M[_], A](protected val ma: M[A]) extends AnyVal {
+  def parReplicateA(n: Int)(implicit P: Parallel[M]): M[List[A]] =
+    Parallel.parReplicateA(n, ma)
+}
+
+final class NonEmptyParallelApOps[M[_], A](protected val ma: M[A]) extends AnyVal {
```

Comment:
`protected val ma: M[A]` – I would suggest to keep everything `private` unless it is necessary to relax the access for some reason.